src/Router.tsx

```diff
@@ -37,63 +38,84 @@ const V2CreateRoute = () => (
   </V2EntryGuard>
 )
 
-export default function Router() {
+function usePageViews() {
+  let location = useLocation()
+
+  useEffect(() => {
+    window.fathom?.trackPageview({
+      url: location.pathname,
+    })
+  }, [location])
+}
+
+function JuiceboxSwitch() {
+  usePageViews()
   const isV2Enabled = featureFlagEnabled(FEATURE_FLAGS.ENABLE_V2)
 
   return (
-    <HashRouter>
-      <Switch>
-        <Route exact path="/">
-          <V1CurrencyProvider>
-            <Landing />
-          </V1CurrencyProvider>
+    <Switch>
```

Comment:
Nothin changed here, just moved Switch to it's own component so we can use the usePageViews hook above it (but with HashRouter)